0000-backtrace-in-core.md

```diff
@@ -0,0 +1,145 @@
+- Feature Name: Backtrace in `core`
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+This RFC will move the `Backtrace` type to `core`, making it available for usage in `no_std` contexts. This is a stepping stone for the `Error` type in `core` and eventually both types will end there. 
+
+It is still unclear whether `Backtrace` itself does need to be moved to `core` and in time this RFC will be unanimous in this matter.
+
+# Motivation
+[motivation]: #motivation
+
+The main reason behind moving `Backtrace` to `core` is to have essential types available for wider usage without the need to import `std`. While `Error` had a valid reason for not being in `core` (it relies on `Box` type for different conversions), `Backtrace` does not have similar blockers apart from its frame-allocating API
```

Comment:
should the tradeoffs not be mentioned in the drawbacks section?

Reply:
I think there needs to be some high-level discussion of the different kinds of solutions up front, otherwise its hard to understand why there isn't a single proposal. Drawbacks section should include a more thorough, detailed listing of drawbacks of the proposal at a high level and the specific solutions (IMO).

Reply:
Please check if it looks good now.